source/reference/ejson.txt

```diff
@@ -32,6 +32,11 @@ to transform BSON data.
      - Convert BSON objects to strings. This method is useful to
        transform ``mongosh`` output.
 
+   * - :ref:`EJSON.serialize() <mongosh-ejson-serialize>`
+     - Convert BSON objects to Extended JSON strings and then reparse
+       the strings as JSON objects. This method is useful to export JSON
```

Comment:
How this works under the hood is an implementation detail, I'd maybe rather explain what it does rather than how it does it, something alone the lines of

```suggestion
     - Convert BSON objects to their Extended JSON representations as 
        JavaScript objects. This method is useful to export JSON
```

Conceptually, users should be thinking of `EJSON.stringify()` as `EJSON.serialize()` + `JSON.stringify()`, not of `EJSON.serialize()` as `EJSON.stringify()` + `JSON.parse()`.

Reply:
Done.